ui/src/app/views/concept-add-modal.tsx

```diff
@@ -125,9 +125,10 @@ export const ConceptAddModal = withCurrentWorkspace()
         addedIds: conceptIds
       };
       try {
-        await conceptSetsApi().createConceptSet(namespace, id, request);
+        const createdConceptSet =
+          await conceptSetsApi().createConceptSet(namespace, id, request);
         this.setState({saving: false});
-        onSave();
+        onSave(createdConceptSet);
```

Comment:
The concept set also needs to be passed to the `onSave()` on line [113](https://github.com/all-of-us/workbench/pull/2174/files#diff-d0aaa77339d78f2f09ba5efb57acb13aR113). Currently it throws an error if I try to add to an existing set.

Reply:
Good catch. That actually maybe means that saveNew and updateExisting should be different. Will check product behavior with Keri.